test/mix/tasks/phx.gen.auth_test.exs

```diff
@@ -192,16 +192,16 @@ defmodule Mix.Tasks.Phx.Gen.AuthTest do
 
       assert_file("lib/my_app_web/components/layouts/root.html.heex", fn file ->
         assert file =~
-                 ~S|<.link href={~p"/users/settings"}>Settings</.link>|
+                 ~r|<.link href={~p"/users/settings"}.*>Settings</.link>|
```

Comment:
Rather than duplicate the tailwind classes in all of these assertions, I shifted the assertions to regexes to be more permissive to tailwind style changes down the road.

The longer, multi-line snapshot tests (below) have been updated with explicit matches.